src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
 template: upgrade
 sidebar: true
 image: ../../../assets/upgrades/merge.png
-summaryPoint1: Eventually the current Ethereum Mainnet will "merge" with the beacon chain proof-of-stake system.
+summaryPoint1: Soon the current Ethereum Mainnet will merge with the Beacon Chain proof-of-stake system.
 summaryPoint2: This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.
-summaryPoint3: This is planned to precede the roll out of shard chains.
-summaryPoint4: We formerly referred to this as "the docking."
+summaryPoint3: This sets the stage for future scaling upgrades including data sharding.
+summaryPoint4: With The Merge, Ethereum's energy consumption drops by 99.95%. Welcome to a new greener Ethereum.
 ---
 
 <UpgradeStatus dateKey="page-upgrades-merge-date">
-  This upgrade represents the official switch to proof-of-stake consensus. This eliminates the need for energy-intensive mining, and instead secures the network using staked ether. A truly exciting step in realizing the <a href="/upgrades/vision/">Ethereum vision</a> – more scalability, security, and sustainability.
+The Merge is the most significant and high-stakes upgrade in the history of Ethereum. Extensive testing and bug bounties were undertaken to ensure a safe transition to proof-of-stake.
```

Comment:
I like this update but I also think the previous explanation was pretty solid:

> This eliminates the need for energy-intensive mining, and instead secures the network using staked ether. A truly exciting step in realizing the Ethereum vision – more scalability, security, and sustainability.

I wonder if there's an aspect of that we could keep? Perhaps down in the explanation below?